app/src/main/java/com/example/simplegithubapp/main/OrganizationAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.simplegithubapp.main
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.simplegithubapp.R
+
+class OrganizationAdapter(private val context:Context):RecyclerView.Adapter<OrganizationHolder>(){
+
+    //data binding로 이미지 받아오기
+    var data = listOf<ImgOrganization>()
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): OrganizationHolder {
+        val view = LayoutInflater.from(context).inflate(R.layout.rv_item_organizations,parent,false)
+        return OrganizationHolder(view, context)
```

Comment:
```suggestion
        val view = LayoutInflater.from(context)
                .inflate(R.layout.rv_item_organizations,parent,false)
        return OrganizationHolder(view, context)
```
.으로 접근하는 메소드들은 한줄에 하나씩 작성하는게 더 가독성이 좋아. 데이터바인딩으로 바꿔보자!

Reply:
엇 이미지는 바인딩 어댑터해야된다고 해서 아직 뷰로 해논 상태!!입니당 언능 공부해서 바꿀게요